src/module/types/actor/Character.ts

```diff
@@ -77,7 +77,7 @@ const CharacterData = () => ({
         "armor", "multi_defense", "reach", "defense", "defense_dodge", "defense_parry",
         "defense_block", "defense_melee", "defense_ranged", "soak", "recoil",
         // Magic/Matrix
-        "drain", "fade", "essence", "public_grid",
+        "drain", "fade", "essence", "public_grid", 'mark_damage',
```

Comment:
Just a thought: 

Adding system modifiers to `system.modifiers` is great! If we find any non modifier values, just flat values, we could make use of the `system.values` structure (that existed at least before the DataModel rework, though was mostly unused).